server/test/users/users.controller.e2e-spec.ts

```diff
@@ -0,0 +1,155 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { userCreateDtoStub } from '../stubs';
+import { UsersController } from '../../src/users/users.controller';
+import { UsersService } from '../../src/users/users.service';
+import { User } from '../../src/users/entities/user.entity';
+import { UsersModule } from '../../src/users/users.module';
+import { getRepositoryToken, TypeOrmModule } from '@nestjs/typeorm';
+import { TEST_DB_OPTIONS } from '../testing-constants';
+import { INestApplication } from '@nestjs/common';
+import * as supertest from 'supertest';
+import { Repository } from 'typeorm';
+
+describe('UsersController', () => {
+  let app: INestApplication;
+  let usersService: UsersService;
+  let repository: Repository<User>;
+
+  let existingRecordId = 0;
+  const seed = {
+    first: 'peter',
+    last: 'parker',
```

Comment:
why aren't these `first_name` and `last_name`? then you could just pass seed as it is instead of having to pass in each property individually

Reply:
if there's some DTO that's blocking this, i vote we just change that DTO